backend/src/Edna/Upload/DB/Query.hs

```diff
@@ -0,0 +1,79 @@
+module Edna.Upload.DB.Query
+  ( insertExperimentFile
+  , insertExperiment
+  , insertSubExperiment
+  , insertMeasurments
+  , insertRemovedMeasurments
+  ) where
+
+import Universum
+
+import Database.Beam.Backend (unSerial)
+import Database.Beam.Postgres (PgJSON(..), Postgres)
+import Database.Beam.Query (default_, insert, insertExpressions, insertValues, val_)
+import Database.Beam.Query.Internal (QExpr)
+
+import Edna.DB.Integration (runInsert', runInsertReturningList', runInsertReturningOne')
+import Edna.DB.Schema
+  (EdnaSchema(..), ExperimentT(..), MeasurementT(..), RemovedMeasurementsT(..), SubExperimentT(..),
+  ednaSchema, theOnlyAnalysisMethodId)
+import Edna.ExperimentReader.Types as EReader
+import Edna.Setup (Edna)
+import Edna.Upload.DB.Schema (ExperimentFileT(..))
+
+insertExperiment :: Word32 -> Word32 -> Word32 -> Edna Word32
```

Comment:
Erm, this type looks very said, it has 3 `Word32` arguments that are easy to confuse and returns `Word32` as well. Let's use `CompoundId` and `TargetId` please. In #42 I am adding `ExperimentId`, let's see what will be merged first.

Reply:
I also added `ExperimentId`, looks like it will be easy to rebase it